tests/utils/test_filter.py

```diff
@@ -2,9 +2,86 @@
 Tests filter functions in filip.utils.filter
 """
 import unittest
+from datetime import datetime
+
+from filip.clients.ngsi_v2 import ContextBrokerClient
+from filip.models.base import FiwareHeader
+from filip.models.ngsi_v2.base import EntityPattern
+from filip.models.ngsi_v2.subscriptions import Subscription
+from filip.utils import filter
+from filip.utils.cleanup import clear_all
+from tests.config import settings
 
 
 class TestFilterFunctions(unittest.TestCase):
```

Comment:
please have a look at utils.iot there you will find the other filter I mentioned

Reply:
Then I think the best place for this testcase is in `tests.clients.test_ngsi_v2_cb.py`. What do you think? Or it should be separate in `tests.utils`?